### PR TITLE
acpi: Allow firmware path configuration

### PR DESCRIPTION
**Pull request by AdamCDunlap**
The Google hypervisor uses ACPI tables that are at a different path. The RSDP table also has extra information tacked onto the end. Allow passing the paths to these tables as environment variables and allow the RSDP table to be larger than expected.

### CODE REVIEW

**Comment by deeglaze** [comment]
Context is that this is for ACPI v2.0 support through a different method than searching the EFI_SYSTEM_TABLE, so the etc/acpi/{tables,rsdp} paths for v1.0 are inadequate.

**Comment by 00xc** [comment]
This should be documented in [INSTALL.md](https://github.com/coconut-svsm/svsm/blob/2d19d4fa150f0a36ef4b28d9fa5e6cd0789835e7/Documentation/INSTALL.md).

**Comment by AdamCDunlap** [comment]
> This should be documented in [INSTALL.md](https://github.com/coconut-svsm/svsm/blob/2d19d4fa150f0a36ef4b28d9fa5e6cd0789835e7/Documentation/INSTALL.md).

Thanks. I added a note there.

**Review by 00xc — APPROVED** [review]

File: Documentation/INSTALL.md [hunk 1]
```diff
@@ -224,6 +224,15 @@ Unit tests can be run inside the SVSM by
 $ QEMU=/path/to/qemu OVMF=/path/to/firmware/ make test-in-svsm
 ```
 
+Different (non-QEMU) hypervisors may provide the ACPI tables and ACPI RSDP at
+different paths. If this is the case, they can be provided as environment
+variables, e.g.
+```
+$ ACPI_RSDP_PATH=etc/acpi/rsdp ACPI_TABLES_PATH=etc/acpi/tables FW_FILE=/path/to/firmware/OVMF.fd make
```
Comment (stefano-garzarella) [inline Documentation/INSTALL.md:231]: I'd suggest adding in the documentation the default paths and maybe use something different in the example.
The rest LGTM.

**Review by stefano-garzarella — APPROVED** [review]
Maybe you can merge the last 2 commits together, however the final result LGTM.

**Comment by AdamCDunlap** [comment]
> Maybe you can merge the last 2 commits together, however the final result LGTM.

Done. Thanks for the review!

**Review by stefano-garzarella — APPROVED** [review]

**Comment by joergroedel** [comment]
CI is failing. Please update the PR so that the commit includes a commit message and make CI pass.

**Comment by AdamCDunlap** [comment]
Fixed the commit message so CI is passing now.

**Review by 00xc — APPROVED** [review]

**Pull request merged**